component-library/src/app/components/code-viewer/code-viewer.component.ts

```diff
@@ -108,3 +108,17 @@ export class codeViewerComponent implements OnInit, OnChanges {
     this.height = height;
   }
 }
+
+/**
+ * Convert config object to string
+ */
+export function stringify(object: any): string {
```

Comment:
I assume you checked with Alexandre to see how the library works?

Reply:
We went over the component/function. For this use case, we need some form of JSON object parsing, and this solution works.

Reply:
It's also nice since the code will be reusable for the other components.